huaweicloud/services/vpc/resource_huaweicloud_vpc_subnet.go

```diff
@@ -28,12 +28,13 @@ var defaultDNSList = map[string][]string{
 	"cn-south-1":     {"100.125.1.250", "100.125.136.29"},  // Guangzhou
 	"cn-southwest-2": {"100.125.1.250", "100.125.129.250"}, // Guiyang-1
 	"ap-southeast-1": {"100.125.1.250", "100.125.3.250"},   // Hong Kong
-	"ap-southeast-2": {"100.125.1.250", "182.50.80.4"},     // Bangkok
+	"ap-southeast-2": {"100.125.1.250", "100.125.1.251"},   // Bangkok
```

Comment:
pls replace the defaultDNSList variable with API:  https://apiexplorer.developer.huaweicloud.com/apiexplorer/debug?product=DNS&api=ListNameServers

Reply:
I am not sure the API can be called in all regions and the API response does not including all regions, for example, **la-north-2**, so we should keep using the map.